src/rust/engine/process_execution/src/local.rs

```diff
@@ -36,6 +36,7 @@ pub struct CommandRunner {
   pub store: Store,
   executor: task_executor::Executor,
   work_dir_base: PathBuf,
+  named_caches: Option<NamedCaches>,
```

Comment:
Why an `Option`? Would hold off on that until some kind of configurability is added